2023.04-4week/ukss/[JS] BOJ.2563.색종이/색종이.js

```diff
@@ -0,0 +1,24 @@
+const list = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');
+
+let cnt = list.shift();
+let area = 0;
+let arr = Array.from(Array(100), () => Array(100).fill(false));
+
+for (let i = 0; i < cnt; i++) {
+  let [x, y] = list[i].split(' ').map(Number);
```

Comment:
```suggestion
let cnt = +list.input[0];
let area = 0;
let arr = Array.from(Array(100), () => Array(100).fill(false));

for (let i = 1; i <= cnt; i++) {
  let [x, y] = list[i].split(' ').map(Number);
```

  - shift를 쓰기보다 input[0]으로 cnt를 가져오고 for문 안에서는 인덱스 1번부터 시작하면 동일한 결과를 얻을 수 있을 것 같아요!
  - shift가 배열의 첫 번째 요소를 제거하는 것이라 리소스가 큰 메서드라고 합니다!👍🏻

Reply:
shift 보다 리소스를 덜 사용할 수 있는 방법이군요 ! 더 좋은 코드로 다듬어 봐야겠습니다 !